src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -1,33 +1,45 @@
-import React, { useState } from 'react'
-import { GraphsHeader } from './GraphsHeader'
-import { HomeLink } from '../../HomeLink'
-import { SectionFAQ } from './SectionFAQ.jsx'
-import { SectionFilerInfo } from './SectionFilerInfo'
-import { SectionGraphs } from './SectionGraphs'
-import { SectionSelector } from '../SectionSelector'
-import Error from '../../../common/Error'
-import '../graphs.css'
+import React, { useEffect, useState } from "react"
+import { GraphsHeader } from "./GraphsHeader"
+import { HomeLink } from "../../HomeLink"
+import { SectionFAQ } from "./SectionFAQ.jsx"
+import { SectionFilerInfo } from "./SectionFilerInfo"
+import { SectionGraphs } from "./SectionGraphs"
+import { SectionSelector } from "../SectionSelector"
+import { Switch, Route } from "react-router-dom"
+import Error from "../../../common/Error"
+import "../graphs.css"
 
-const SectionOptions = ['Graphs', 'Filer Info', 'FAQ']
+const SectionOptions = [
+  { label: "Graphs" }, // url for graphs comes from redux
+  { label: "Filers' Info", url: "/filers" },
+  { label: "FAQ", url: "/faq" },
+]
 
-export const QuarterlyGraphs = props => {
+export const QuarterlyGraphs = (props) => {
   const [error, setError] = useState()
   const [graphHeaderOverview, setGraphHeaderOverview] = useState() // Populated from API
   const [section, setSection] = useState(SectionOptions[0])
 
+  useEffect(() => {
+    // Handling which tab is active via url
+    if (props.history.location.pathname.includes("/info/filers")) {
+      setSection(SectionOptions[1])
+    } else if (props.history.location.pathname.includes("/info/faq")) {
+      setSection(SectionOptions[2])
+    } else {
+      setSection(SectionOptions[0])
+    }
+  }, [props.history.location.pathname])
+
   return (
-    <div className='Graphs'>
+    <div className="Graphs">
       <HomeLink />
       <GraphsHeader overview={graphHeaderOverview} />
       <Error error={error} />
-      <SectionSelector
-        options={SectionOptions}
-        selected={section}
-        onChange={setSection}
-      />
-      <div className='section-wrapper'>
+      <SectionSelector options={SectionOptions} selected={section} />
```

Comment:
`<SectionSelector>` should be able to internally determine which section is selected.